services/QuillLMS/app/models/admin_report_filter_selection.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: admin_report_filter_selections
+#
+#  id                :bigint           not null, primary key
+#  filter_selections :jsonb
+#  report            :string           not null
+#  created_at        :datetime         not null
+#  updated_at        :datetime         not null
+#  user_id           :bigint           not null
+#
+# Indexes
+#
+#  index_admin_report_filter_selections_on_report   (report)
+#  index_admin_report_filter_selections_on_user_id  (user_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (user_id => users.id)
+#
+class AdminReportFilterSelection < ApplicationRecord
+  belongs_to :user
+
+  validates :user_id, presence: true
+
```

Comment:
What about using `serialize :filter_selections, JSON`?

For example https://blog.kiprosh.com/serialization_in_ruby_on_rails_part_two

The attributes are automatically serialized and deserialized by ActiveRecord, allowing you to work with them in Ruby as if they were regular attributes which can simplify application logic.